skipper.go

```diff
@@ -1203,9 +1200,19 @@ func listenAndServeQuit(
 	if err != nil {
 		return err
 	}
+	serveTLS := tlsConfig != nil
+
+	address := o.Address
+	if address == "" {
+		if serveTLS {
+			address = ":https"
+		} else {
+			address = ":http"
+		}
+	}
```

Comment:
Moved out empty HTTP address handling from `listen` and added empty HTTPS address handling as [ListenAndServeTLS does](https://github.com/golang/go/blob/6244b1946bc2101b01955468f1be502dbadd6807/src/net/http/server.go#L3272-L3274).